### DISCUSSION TITLE
Welcome to Wheels Discussions!

### DISCUSSION THREAD

**Discussion by bpamiri** [wheels-dev/wheels#1117] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by timsayshey** [comment]
Hey guys - I'm Tim. I designed the CFWheels logo with Tom during the revival back in 2015. I'm a huge CFWheels fan, and used to contribute to the mailing list back in the day. I built WheelieCMS, a Wordpress inspired CMS built on CFWheels. I also built CFWheels extensions for SublimeText and Dreamweaver. These days I'm doing less CFWheels because I'm a full-time frontend engineer doing VueJS work. However I still maintain WheelieCMS and use it in my side project, a growing SaaS product. Looking forward to the bright future of CFWheels! Thanks for pushing the project forward Peter

**Reply by bpamiri** [reply]
Tim,

Great to hear from you. Have you looked at HTMX.org? Since you’re doing more front end work I was wondering what you think of htmx for building dynamic front ends with backend tech.
